Purchases/Logging/Logger.swift

```diff
@@ -14,57 +14,42 @@
 
 import Foundation
 
-@objc(RCLogLevel) public enum LogLevel: Int {
+@objc(RCLogLevel) public enum LogLevel: Int, CustomStringConvertible {
 
     case debug, info, warn, error
 
-    func description() -> String {
+    public var description: String {
```

Comment:
this needs to be public now, which I don't think is a bad thing - it'll allow folks using their own log handler to use our string descriptions of the enum value if they'd like.

Reply:
//cc @beylmk, just to note that this will be made public in the next major version

Reply:
Added to migration doc